media/js/base/global.js

```diff
@@ -87,15 +87,16 @@ function getFirefoxMasterVersion(userAgent) {
     return version;
 }
 
+// Used on the plugincheck page to also support all browsers based on Gecko.
+function isLikeFirefox(userAgent) {
+    var ua = userAgent || navigator.userAgent;
+    return /Iceweasel/i.test(ua) || /IceCat/i.test(ua)
+        || /SeaMonkey/i.test(ua) || /Camino/i.test(ua);
+}
+
 function isFirefox(userAgent) {
     var ua = userAgent || navigator.userAgent;
-    // camino UA string contains 'like Firefox'
-    return (
-        (/\sFirefox/).test(ua) &&
-        !(/like Firefox/i).test(ua) &&
-        !(/Iceweasel/i).test(ua) &&
-        !(/SeaMonkey/i).test(ua)
-    );
+    return (/\sFirefox/).test(ua) && !isLikeFirefox(ua);
```

Comment:
Scratch that, looks like the parens are needed for old Gecko versions: http://mdn.beonex.com/en/Core_JavaScript_1.5_Reference/Global_Objects/RegExp/test.html

We should stick to using them here.